test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -151,31 +161,36 @@ TEST_F(SynchronizerTest, ValidWhenBadStorage) {
 
 TEST_F(SynchronizerTest, ValidWhenBlockValidationFailure) {
   // commit from consensus => chain validation failed => commit successful
-  iroha::model::Block test_block;
-  test_block.height = 5;
-  test_block.sigs.emplace_back();
-  test_block.hash = iroha::hash(test_block);
+  TemplateMockBlockValidator<MockBlockValidator> mockBlockValidator;
+  EXPECT_CALL(*mockBlockValidator.validator, validate(_))
+      .WillOnce(Return(shared_model::validation::Answer()));
+  using TestUnsignedBlockBuilder = shared_model::proto::TemplateBlockBuilder<
```

Comment:
Maybe move out to test builders?

Reply:
It's possible but to do that we also have to move all Mock there what is not a good idea I think.